pkg/apis/kubeone/types.go

```diff
@@ -935,6 +938,43 @@ type StaticAuditLogConfig struct {
 	LogMaxSize int `json:"logMaxSize,omitempty"`
 }
 
+type WebHookAuditLog struct {
```

Comment:
The API type should be called `WebhookAuditLog` (with lowercase `h`). That's also to make it consistent with different Kubernetes resources, e.g. `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook`